### ISSUE TITLE
Bug on PTR 

### ISSUE THREAD

**Issue by robisaks** [mooreatv/MoLib#19]
Tested on PTR for 8.3. I was specifically trying to test the Auction House scanning of the library, but the bug catch happens as soon as you log in.

> Message: Interface\AddOns\AuctionDB\MoLib/MoLib.lua:938: Attempt to register unknown event "AUCTION_ITEM_LIST_UPDATE"
Time: Sun Jan  5 12:04:18 2020
Count: 1
Stack: Interface\AddOns\AuctionDB\MoLib/MoLib.lua:938: Attempt to register unknown event "AUCTION_ITEM_LIST_UPDATE"
[string "=[C]"]: in function `RegisterEvent'
[string "@Interface\AddOns\AuctionDB\MoLib/MoLib.lua"]:938: in function `RegisterEventHandlers'
[string "@Interface\AddOns\AuctionDB\AuctionDB.lua"]:343: in main chunk
Locals: (*temporary) = <unnamed> {
 0 = <userdata>
 name = "AHDB"
 addonPtr = <table> {
 }
}
(*temporary) = "AUCTION_ITEM_LIST_UPDATE"

**Comment by mooreatv** [comment]
yeah I have focused on classic and haven't looked yet at bfa/sl for the AH revamp sorry
